ref/Boolean_Operators.md

```diff
@@ -181,7 +181,14 @@ This operator is similar to **&gt;** but tests for **less than or equal to**.
 
 ------
 
-#####  Fuzzy comparisons: `~=`, `~!=`, `~<`, `~>`, `~>=`, `~<=`
+####  Fuzzy comparisons:
+
+##### Approximately equal: `‹expr1› ~= ‹expr2›`
+##### Not approximately equal: `‹expr1› ~!= ‹expr2›`
```

Comment:
And I would really stick with 'approximately not equal', etc. The picture from p. 254 of the printed book might be helpful as well.